### PR TITLE
Add the --dry-run option to avoid sending batch requests.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: kinto_wizard/__main__.py [hunk 1]
```diff
@@ -55,7 +58,8 @@ def main():
 
     thread_pool = ThreadPoolExecutor()
     event_loop = asyncio.get_event_loop()
-    async_client = AsyncKintoClient(client, event_loop, thread_pool)
+    async_client = AsyncKintoClient(client, event_loop, thread_pool,
+                                    dry_run=getattr(args, 'dry_run', False))
```
Comment (leplatrem) [inline kinto_wizard/__main__.py]: wouldn't it be nice to set a default at the subparser level ?

File: kinto_wizard/async_kinto.py [hunk 2]
```diff
@@ -8,34 +8,40 @@
 """
 
 import functools
+import mock
```
Comment (leplatrem) [inline kinto_wizard/async_kinto.py]: mock is now part of stdlib ;)

File: setup.py [hunk 3]
```diff
@@ -17,7 +17,8 @@ def read_file(filename):
 
 REQUIREMENTS = [
     "kinto-http>=8",
-    "ruamel.yaml"
+    "ruamel.yaml",
+    "mock",  # We use it for --dry-run
```
Comment (leplatrem) [inline setup.py]: as said above: mock is now in unittest stdlib ;)

**Comment by leplatrem** [comment]
Please add a changelog entry please

File: kinto_wizard/__main__.py (hunk 1 above)
Comment (Natim) [reply kinto_wizard/__main__.py]: Actually it didn't make sense to add a --dry-run for the dump command, and we have a default for the load command.

File: kinto_wizard/async_kinto.py (hunk 2 above)
Comment (Natim) [reply kinto_wizard/async_kinto.py]: Yes I tried to find it didn't know it was encapsulated inside unittest

**Pull request merged**